source/api/discovery/0.3/index.md

```diff
@@ -0,0 +1,784 @@
+  "type": "Update",
+  "summary": "admin updated the manifest, fixing reported bug #15.",
+  "object": {
+    "id": "https://example.org/iiif/1/manifest",
+    "type": "Manifest",  	
+    "seeAlso": [
+      {
+        "id": "https://example.org/dataset/single-item.jsonld",
+        "type": "Dataset",
+        "format": "application/ld+json"
+      }
+    ]
+  },
+  "endTime": "2017-09-21T00:00:00Z",
+  "startTime": "2017-09-20T23:58:00Z",
+  "actor": {
+    "id": "https://example.org/person/admin1",
+    "type": "Person"
+  }
+}
+```
+
+
+### 3.4. Linked Data Context and Extensions
+
+#### 3.4.1. @context
+
+The top level resource in the response _MUST_ have the `@context` property, and it _SHOULD_ appear as the very first key/value pair of the JSON representation.  This property lets Linked Data processors interpret the document as a graph.  The value of the property _MUST_ be either the URI of the IIIF Discovery context document, `http://iiif.io/api/discovery/0/context.json`, or it _MUST_ be an array of strings, where the URI of the IIIF Discovery context document is the last item in the array.  
+
+```
+{
+  "@context": "http://iiif.io/api/discovery/0/context.json"
+}
+```
+
+#### 3.4.2. Extensions
+
+If any additional classes or properties are desired beyond the ones defined in this specification or the ActivityStreams specification, then those classes or properties _SHOULD_ be mapped to RDF terms in one or more additional context documents. These extension contexts _SHOULD_ be added to the top level `@context` property, and _MUST_ be before the discovery context. The JSON-LD 1.1 functionality of defining terms only within a specific property, known as [scoped contexts][org-w3c-json-ld-scoped-contexts], _MUST_ be used to minimize cross-extension collisions.  Extensions intended for broad use _SHOULD_ be registered in the [extensions registry][annex-registry].  
+
+```
+{
+  "@context": [
+    "http://example.org/extension/context.json",
+    "http://iiif.io/api/discovery/0/context.json"
+  ]
+}
+```
+
+
+### 3.5. Activity Streams Processing Algorithm
+{: #activity-streams-processing-algorithm}
+
+The aim of the processing algorithm is to inform consuming applications how to make best use of the available information.
+
+#### 3.5.1. Collection Algorithm
+{: #collection-algorithm}
+
+Given the URI of an ActivityStreams Collection (`collection`) as input, a conforming processor _SHOULD_:
+
+<!-- This should be nested * markdown... but can't figure it -->
+
+<ol>
+  <li>Initialization:
+    <ol>
+      <li>Let <code class="highlighter-rouge">processedItems</code> be an empty array</li>
+      <li>Let <code class="highlighter-rouge">lastCrawl</code> be the timestamp of the previous time the algorithm was executed</li>
+    </ol>
+  </li>
+  <li>Retrieve the representation of <code class="highlighter-rouge">collection</code> via HTTP(S)</li>
+  <li>Minimally validate that it conforms to the specification</li>
```

Comment:
Wondering what "Minimally validate" means... "At least validate" or "Partially validate"? Would striking "Minimally" be clearer?

Reply:
It's intended as a warning to validate the input at least such that it's JSON and approximately correct structure. It could be omitted completely.  But no need to validate all the extensions, and so forth. Replaced with:

> Validate that the retrieved representation contains at least the features required for processing

Reply:
:+1: